java/fury-core/src/main/java/org/apache/fury/memory/MemoryBuffer.java

```diff
@@ -156,8 +156,17 @@ private MemoryBuffer(long offHeapAddress, int size, ByteBuffer offHeapBuffer) {
    *     the memory being released.
    * @param streamReader a reader for reading from a stream.
    */
-  private MemoryBuffer(
+  public MemoryBuffer(
```

Comment:
We'd better keep it private, the MemoryBuffer should only be created by factory method